controllers/kubeobjects.go

```diff
@@ -47,84 +49,111 @@ func kubeObjectsRecover(
 		objectWriter{ctx: ctx, Writer: writer, log: log},
 		reader,
 		s3Url,
 		s3BucketName,
 		s3KeyPrefix,
-		sourceNamespaceName,
+		sourceNamespacedName.Namespace,
 		targetNamespaceName,
+		restoreSpec,
+		captureNumber,
 	)
 }
 
 func backupDummyCreateAndIfAlreadyExistsThenRestore(
-	namespacedName types.NamespacedName,
+	restoreNamespacedName types.NamespacedName,
+	backupNamespacedName types.NamespacedName,
 	w objectWriter,
 	reader client.Reader,
 	s3Url string,
 	s3BucketName string,
 	s3KeyPrefix string,
 	sourceNamespaceName string,
 	targetNamespaceName string,
+	restoreSpec ramendrv1alpha1.ResourceRestoreInstance,
+	captureNumber int64,
 ) error {
 	backupLocation, backup, err := w.backupCreate(
-		namespacedName, s3Url, s3BucketName, s3KeyPrefix, backupSpecDummy(), sourceNamespaceName,
+		backupNamespacedName, s3Url, s3BucketName, s3KeyPrefix, backupSpecDummy(), sourceNamespaceName, captureNumber,
 	)
 	if err != nil {
 		return err
 	}
 
-	if err := reader.Get(w.ctx, namespacedName, backup); err != nil {
+	backupObjectNamespacedName := types.NamespacedName{
+		Name:      backupNamespacedName.Name,
+		Namespace: VeleroNamespaceNameDefault,
+	}
+	if err := reader.Get(w.ctx, backupObjectNamespacedName, backup); err != nil {
 		return pkgerrors.Wrap(err, "dummy backup get")
 	}
 
-	return backupDummyStatusProcess(backupLocation, backup, namespacedName, w, reader,
+	// checks status of dummy backup and possibly starts restore
+	return backupDummyStatusProcess(backupLocation, backup, backupNamespacedName,
+		restoreNamespacedName,
+		w, reader,
 		sourceNamespaceName,
 		targetNamespaceName,
+		restoreSpec,
 	)
 }
 
 func backupDummyStatusProcess(
 	backupLocation *velero.BackupStorageLocation,
 	backup *velero.Backup,
-	namespacedName types.NamespacedName,
+	backupNamespacedName types.NamespacedName,
+	restoreNamespacedName types.NamespacedName,
 	w objectWriter,
 	reader client.Reader,
 	sourceNamespaceName string,
 	targetNamespaceName string,
+	restoreSpec ramendrv1alpha1.ResourceRestoreInstance,
 ) error {
 	backupStatusLog(backup, w.log)
 
+	w.log.Info(fmt.Sprintf("backupDummyStatusProcess: %s Status.Phase=%s", backup.Name, backup.Status.Phase))
+
 	switch backup.Status.Phase {
 	case velero.BackupPhaseCompleted:
 		// deletes backup and backup object TODO delete bsl also after both are gone?
-		return w.objectCreate(backupDeletion(namespacedName))
+		// disabled so successful backup status from existing object doesn't delete contents
+		// return w.objectCreate(backupDeletion(backupNamespacedName))
+		fallthrough
 	case velero.BackupPhasePartiallyFailed:
 		fallthrough
 	case velero.BackupPhaseFailed:
 		// TODO if failed because backup already exists
-		return backupRestore(backupLocation, backup, namespacedName, w, reader, sourceNamespaceName, targetNamespaceName)
+		w.log.Info(fmt.Sprintf("backupDummyStatusProcess: BackupPhaseFailed on '%s'. Creating restore '%s'",
+			backupNamespacedName.Name, restoreNamespacedName.Name))
```

Comment:
(nit) `log.Info` likes an odd number or arguments.  It expects an initial string, followed by string, value pairs, where the string names the subsequent value, e.g.:
```go
w.log.Info("backupDummyStatusProcess: BackupPhaseFailed", "backup", backupNamespacedName, "restore",  restoreNamespacedName)
```